### DISCUSSION TITLE
REST API 란 무엇인가

### DISCUSSION THREAD

**Discussion by starshb** [starshb/pharmacy_Find#17] [Q&A] [answered]
REST API 란 무엇인가

**Comment by tjdud22** [comment, marked as answer]
REST API(REpresentational State Transfer)는 웹상에서 사용되는 여러 리소스를 HTTP URI로 표현하고, 해당 리소스에 대한 행위를 HTTP Method로 정의하는 방식을 말합니다.

쉽게 말해서,
리소스(HTTP URI로 정의됨)를 어떻게 하겠다(HTTP Method + Payload)를 구조적으로 깔끔하게 표현하는 방법입니다.

**Comment by tjdud22** [comment]
rest api 란?
RESTful한 API를 말하며 일련의 특징과 규칙등을 지키는 API를 일컫는다.
